nltk/treeprettyprinter.py

```diff
@@ -93,6 +93,8 @@ def __init__(self, tree, sentence=None, highlight=()):
                         for n, b in enumerate(a):
                             if not isinstance(b, Tree):
                                 a[n] = len(sentence)
+                                if type(b) == tuple:
+                                    b = '/'.join(b)
```

Comment:
These type checking is awkward but I guess it's necessary. 

@dimazest any suggestions?

Reply:
@alvations would ```isinstance``` be better than ```type```?